src/components/DrawerBar.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { Button, Drawer, List, ListItem, ListItemButton, ListItemIcon, ListItemText, Typography } from '@mui/material';
+import Avatar from '@mui/material/Avatar';
+import image1 from '../imgs/image1.png'
+import MovingIcon from '@mui/icons-material/Moving';
+import AccountBoxIcon from '@mui/icons-material/AccountBox';
+import TimelineIcon from '@mui/icons-material/Timeline';
+import './drawerbar.css'
+
+const DrawerBar = () => {
+    const logout = () => {
+        // do a logout
+    }
+
+    return (
+        <Drawer variant="permanent"
+            sx={{
+                flexShrink: 0,
+                '& .MuiDrawer-paper': {
+                    overflow: 'hidden',
+                    position: 'relative',
+                    width: '350px',
+                    height: '600px'
+                },
+            }}
+            open
+        >
+            <Avatar
+                alt='user profile picture'
+                src={image1}
+                className='picture'
+                sx={{
+                    width: 100,
+                    height: 100,
+                }}
+            />
+            <Typography variant='h4' sx={{ position: 'relative', top: 70, left: 120 }}>John Doe</Typography>
+            <Typography
+                sx={{
+                    position: 'relative',
+                    top: 80,
+                    left: 140,
+                    fontSize: 12,
+                    color: 'rgba(0, 0, 0, 0.38)',
+                }}>Joined 10/06/2022</Typography>
+            <List className='list'>
+                <ListItem disablePadding>
+                    <ListItemButton>
+                        <ListItemIcon>
+                            <MovingIcon />
+                        </ListItemIcon>
+                        <ListItemText primary='Transaction History' />
+                    </ListItemButton>
+                </ListItem>
+                <ListItem disablePadding>
+                    <ListItemButton>
+                        <ListItemIcon>
+                            <TimelineIcon />
+                        </ListItemIcon>
+                        <ListItemText primary='Currency ratio history' />
+                    </ListItemButton>
+                </ListItem>
+                <ListItem disablePadding>
+                    <ListItemButton>
+                        <ListItemIcon>
+                            <AccountBoxIcon />
+                        </ListItemIcon>
+                        <ListItemText primary='Profile' />
+                    </ListItemButton>
+                </ListItem>
+            </List>
+            <Button
+                type='button'
+                variant='contained'
+                size='large'
+                className='logoutButton'
+                onClick={logout}
+            >
+                Log out
+            </Button>
+        </Drawer>
+    )
+}
+
+export default DrawerBar
```

Comment:
nit picking: You don't need to have it as a default you can just export as a named component and import it accordingly. This way you will have a more tidy code. Right now you can import this component with whatever name you want which is fine for this particular case but in big applications can become quite hard to maintain. 
`export const DrawerBar = ()=>{....}` , `import {DrawerBar} from ...`

Reply:
Consider this also for the moments when you might have multiple components/methods in one file and you might need them in other places.